less/components/checkbox.less

```diff
@@ -0,0 +1,77 @@
+.Checkbox {
+    display: inline-block;
+
+    .Checkbox-wrapper {
+        display: flex;
+        align-items: center;
+        cursor: pointer;
+    }
+
+    // hide the checkbox entirely but keep accessible
+    input[type="checkbox"] {
+        border: 0;
+        clip: rect(0, 0, 0, 0);
+        height: 1px;
+        margin: -1px;
+        overflow: hidden;
+        padding: 0;
+        position: absolute;
+        width: 1px;
+    }
+
```

Comment:
It does work. That's really the reason for those ugly combination of flip / height / margin junk - so you can skin the other element but let the browser still recognize checkbox by not hiding it entirely.